web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@momentum-ui/web-components",
-  "version": "2.1.7",
+  "version": "2.1.8-dev.1",
```

Comment:
Let's make it official with a version patch!

Reply:
yup, so we should make it `2.1.8`?